game.py

```diff
@@ -20,22 +22,43 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
 
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses):
```

Comment:
This compound conditional looks good! One small thing is that this line breaks the PEP8 guidelines for line length (79 characters or less including indentation https://peps.python.org/pep-0008/#maximum-line-length). From the linked guide:

> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses.

It can definitely be tough to break up lines nicely, but when you have compound conditionals, I think the most obvious way to break things up is by the conditional itself. For example, you could do:

```suggestion
    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and \
               not is_word_guessed(snowman_word, correct_letter_guess_statuses):
```

This makes it just a little easier to read!